AmplifyPlugins/API/AWSAPICategoryPluginIntegrationTests/GraphQL/GraphQLWithIAMIntegrationTests/README.md

```diff
@@ -19,46 +19,86 @@ The following steps demonstrate how to set up an GraphQL endpoint with AppSync.
 
 3. `amplify add auth`
 ```perl
-? Do you want to use the default authentication and security configuration? `Default configuration`
-? How do you want users to be able to sign in? `Username`
-? Do you want to configure advanced settings? `No, I am done.`
+Do you want to use the default authentication and security configuration? 
```

Comment:
if these keep changing, could you mention the `amplify --v` version in this document? that way, a developer running these manually can roll back to the previous version of the CLI if really needed